spec/system/claims/support/schools/mentors/support_user_adds_a_mentor_spec.rb

```diff
@@ -272,13 +272,13 @@ def then_i_see_errors(errors)
   end
 
   def then_i_see_form_with_trn(trn)
-    expect(page.find("#claims-mentor-form-trn-field").value).to eq(trn)
+    expect(page.find("#claims-add-mentor-wizard-mentor-step-trn-field").value).to eq(trn)
   end
 
   def then_i_see_form_with_dob(day, month, year)
-    expect(page.find("#claims_mentor_form_date_of_birth_1i").value).to eq(year)
-    expect(page.find("#claims_mentor_form_date_of_birth_2i").value).to eq(month)
-    expect(page.find("#claims_mentor_form_date_of_birth_3i").value).to eq(day)
+    expect(page.find("#claims_add_mentor_wizard_mentor_step_date_of_birth_1i").value).to eq(year)
+    expect(page.find("#claims_add_mentor_wizard_mentor_step_date_of_birth_2i").value).to eq(month)
+    expect(page.find("#claims_add_mentor_wizard_mentor_step_date_of_birth_3i").value).to eq(day)
```

Comment:
It would be nicer to use this:

https://github.com/DFE-Digital/itt-mentor-services/blob/2d7fdfc9b407cdebfe4e5fe8df62b7f8c2b09964/spec/system/claims/schools/mentors/add_a_mentor_spec.rb#L299-L301